src/Illuminate/Mail/Transport/SesTransport.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+namespace Illuminate\Mail\Transport;
+
+use InvalidArgumentException;
+use Symfony\Component\Mailer\Bridge\Amazon\Transport\SesApiAsyncAwsTransport;
+use Symfony\Component\Mailer\Bridge\Amazon\Transport\SesHttpAsyncAwsTransport;
+use Symfony\Component\Mailer\Envelope;
+use Symfony\Component\Mailer\SentMessage;
+use Symfony\Component\Mailer\Transport\TransportInterface;
+use Symfony\Component\Mime\Email;
+use Symfony\Component\Mime\RawMessage;
+
+class SesTransport implements TransportInterface
+{
+    /**
+     * The Amazon SES Symfony transport.
+     *
+     * @var \Symfony\Component\Mailer\Bridge\Amazon\Transport\SesApiAsyncAwsTransport|\Symfony\Component\Mailer\Bridge\Amazon\Transport\SesHttpAsyncAwsTransport
+     */
+    protected $transport;
+
+    /**
+     * The Amazon SES transmission options.
+     *
+     * @var array
+     */
+    protected $options;
+
+    /**
+     * Create a new SES transport instance.
+     *
+     * @param  \Symfony\Component\Mailer\Bridge\Amazon\Transport\SesApiAsyncAwsTransport|\Symfony\Component\Mailer\Bridge\Amazon\Transport\SesHttpAsyncAwsTransport  $transport
+     * @param  array  $options
+     * @return void
+     */
+    public function __construct(TransportInterface $transport, array $options = [])
```

Comment:
Since Laravel 9 requires PHP 8, couldn't the type-hint here be `SesApiAsyncAwsTransport|SesHttpAsyncAwsTransport` instead and remove the exception below?